addon/mixins/model-validator.js

```diff
@@ -47,14 +48,17 @@ export default Ember.Mixin.create({
 
   /**** Validators ****/
   _validateCustom: function(property, validation){
-    var customValidator = this._getCustomValidator(validation);
-    if (customValidator) {
-      var passedCustomValidation = customValidator(property, this.get(property), this);
-      if (!passedCustomValidation) {
-        this.set('isValidNow', false);
-        this._addToErrors(property, validation.custom, Messages.customValidationMessage);
-      }
-    }
+		validation = Ember.isArray(validation.custom) ? validation.custom : [validation.custom];
+		for (var i = 0; i < validation.length; i++) {
+	    var customValidator = this._getCustomValidator(validation[i]);
+	    if (customValidator) {
+	      var passedCustomValidation = customValidator(property, this.get(property), this);
+	      if (!passedCustomValidation) {
+	        this.set('isValidNow', false);
+	        this._addToErrors(property, validation[i], Messages.customValidationMessage);
+	      }
+	    }
+		}
```

Comment:
this now ensures that validation is an array or converts it to one. then iterates over it and executes the custom validation as it did before, but for each in the array